modules/ROOT/pages/snippets/add_channels_mgrsync.adoc

```diff
@@ -0,0 +1,16 @@
+.Procedure: Adding Software Channels at the Command Prompt
+. At the command prompt on the {productname} Server, as root, use the [command]``mgr-sync`` command to add the appropriate channels:
++
+----
+mgr-sync add \
+<base_channel_name> \
+<child_channel_name_1> \
+<child_channel_name_2> \
+... <child_channel_name_n>
+----
+. Synchronize the channels:
++
+----
+mgr-sync sync
+----
```

Comment:
Same problem as in modules/client-configuration/pages/snippets/addchannels_vendor_cli.adoc

Reply:
I've deleted these for now to avoid confusion while we work out the snippets problem.